docs/admin.rst

```diff
@@ -572,6 +574,11 @@ into your Admin Workstation, you should first perform the following troubleshoot
    "This agent has no identities." then you need to add the key via ``ssh-add``
    prior to SSHing into the servers.
 
+#. **Ensure that Ansible is installed.** If Ansible is not installed, 
+   or ``./securedrop-admin tailsconfig`` fails, see
+   :doc:`configure_admin_workstation_post_install` for detailed setup instructions.
```

Comment:
What motivates this addition? We already suggest running `./securedrop-admin tailsconfig` above; the command will instruct the admin to run `./securedrop-admin setup` if the virtual environment has not been initialized. The instructions you link to mainly ask the admin to run `./securedrop-admin tailsconfig`, so this feels somewhat circular.

Reply:
Makes sense. I did pull it from the other troubleshooting steps and put it here as a last resort diagnostic (as in, if none of the above steps work, double-check that ansible is even installed). I wasn't sure how to put that. But if it reads circular we can remove it.

Reply:
Removed this instruction in 151175d